app/livechat/server/api/lib/livechat.js

```diff
@@ -111,7 +111,7 @@ export async function settings() {
 			forceAcceptDataProcessingConsent: initSettings.Livechat_force_accept_data_processing_consent,
 			showConnecting: initSettings.Livechat_Show_Connecting,
 			agentHiddenInfo: initSettings.Livechat_show_agent_info === false,
-			clearLivechatUserSession: initSettings.Livechat_clear_livechat_user_session,
+			removeGuestInfoWhenChatEnded: initSettings.Livechat_remove_guest_info_when_chat_ended,
```

Comment:
The old setting name described better what this feature is supposed to do as the Visitor/Guest won't be deleted when the chat ends, actually just the live chat session will be forgotten/reset.